app/views/cards/show.html.erb

```diff
@@ -32,103 +32,74 @@
+                          <%= avatar_image(user, include_name: true, size: 30) %>
+                        <% end %>
+                      </div>
                     <% end %>
-                  </div>
-                <% end %>
-              <% end %>
+                  <% end %>
 
-            </div>
-          </div>
+                </div>
+              </div>
 
-          <% cache ['card-activity-tab', @card, @activities] do %>
-            <div class="tab-pane" id="activity-tab">
-              <h4 class="header-underline">
-                Recent activity
-                <% if @card.has_revision_history? %>
-                  -
-                  <span class="actions">
-                    <%= link_to(project_board_list_card_revisions_path(current_project, @board, @list, @card)) do %>
-                      <i class="fa fa-history"></i> Revision history
+              <% cache ['card-activity-tab', @card, @activities] do %>
+                <div class="tab-pane" id="activity-tab">
+                  <h4 class="header-underline">
+                    Recent activity
+                    <% if @card.has_revision_history? %>
+                      -
+                      <span class="actions">
+                        <%= link_to(project_board_list_card_revisions_path(current_project, @board, @list, @card)) do %>
+                          <i class="fa fa-history"></i> Revision history
+                        <% end %>
+                      </span>
                     <% end %>
-                  </span>
-                <% end %>
-              </h4>
-              <%= render 'activities/feed', activities: @activities %>
+                  </h4>
+                  <%= render 'activities/feed', activities: @activities %>
+                </div>
+              <% end %>
             </div>
-          <% end %>
-        </div>
-      </div>
-    </div>
-
-    <div class="col-12 col-xxl-5 p-0">
-      <div class="sticky-top">
-        <div class="content-container ml-xxl-3 mt-0">
-          <%= content_tag :div,
-            spinner_tag,
-            data: {
-              behavior: 'fetch fetch-comments',
-              path: comments_path(
-                comment: {
-                  commentable_type: @card.class,
-                  commentable_id: @card.id
-                }
-              )
-            }
-          %>
-        </div>
-
-        <div class="content-container ml-xxl-3">
-          <%= content_tag :div,
-            spinner_tag,
-            data: {
-              behavior: 'fetch fetch-subscriptions',
-              path: subscriptions_path(
-                subscription: {
-                  subscribable_type: @card.class,
-                  subscribable_id: @card.id
-                }
-              )
-            }
-          %>
+          </div>
         </div>
       </div>
     </div>
 
+    <%= render partial: 'shared/resource_sidebar', locals: { resource: @card, content: ['comments', 'subscriptions'] } %>
```

Comment:
What's the reason for passing `content` here? It seems like all calls to this partial are passing the same `content` values. Is there a case where there is a comments feed without subscriptions or vice versa?

Reply:
This is for pro where we will pass the validations section to some views.